src/mon/NVMeofGwSerialize.h

```diff
@@ -17,6 +17,7 @@
 #undef dout_prefix
 #define MODULE_PREFFIX "nvmeofgw "
 #define dout_prefix *_dout << MODULE_PREFFIX << __PRETTY_FUNCTION__ << " "
+#define MAX_SUPPORTED_ANA_GROUPS 16 //used for support backward compatibility
```

Comment:
This is not purely for backward compatibility -- cfg_add_gw restricts the number of ana groups to 16 regardless.

Reply:
fixed in the last push